Orange/widgets/model/tests/test_owneuralnetwork.py

```diff
@@ -21,3 +21,18 @@ def test_migrate_setting(self):
         settings = dict(alpha=103)
         OWNNLearner.migrate_settings(settings, None)
         self.assertEqual(OWNNLearner.alphas[settings["alpha_index"]], 100)
+
+        settings = dict(alpha=0.0001)
```

Comment:
`alpha` does not appear in the `settings` with `version = 1`, so this should be replaced with `settings = dict(alpha_index=0)` which does not pass.

Reply:
Is it possible that alpha is in `version = 1`, but not in a `version is None`? Because now we are at `settings_version = 2`, which was upped from before. And I did not introduce alpha in this version, it was already there.

Reply:
alpha could (probably) be in version = 1, only if the workflow was build upon version = 0, which is not even correct, because alpha is no longer a setting (replaced by alpha_index)